RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -1130,6 +1139,252 @@ private boolean parseSamPluginOutput(String regFilePath, AbstractFile regAbstrac
+                           hostName = parameter.getValue().getAsString();
+                           continue;
+                        }
+                        if (parameter.getName().toLowerCase().equals("domain")) {
+                           domainName = parameter.getValue().getAsString();
+                           continue;
+                        }
+                    }                    
+                } catch (ReadContentInputStreamException ex) {
+                    logger.log(Level.WARNING, String.format("Error reading registry file '%s' (id=%d).",
+                            systemHive.getName(), systemHive.getId()), ex); //NON-NLS
+                    this.addErrorMessage(
+                            NbBundle.getMessage(this.getClass(), "ExtractRegistry.analyzeRegFiles.errMsg.errWritingTemp",
+                            this.getName(), systemHive.getName()));
+                    continue;
+                } catch (RegistryParseException ex) {
+                   continue; 
+                } catch (IOException ex) {
+                    logger.log(Level.SEVERE, String.format("Error writing temp registry file '%s' for registry file '%s' (id=%d).",
+                            systemFileNameLocal, systemHive.getName(), systemHive.getId()), ex); //NON-NLS
+                    this.addErrorMessage(
+                            NbBundle.getMessage(this.getClass(), "ExtractRegistry.analyzeRegFiles.errMsg.errWritingTemp",
+                                    this.getName(), systemHive.getName()));
+                    continue;
+                }
+            }
+        }
+    }
+    
+    /**
+     * Search's a registry hive for the specified key
+     * 
+     * @param registryHiveFile Hive to parse
+     * @param registryKey registry key to find in hive
+     * @return registry key or null if it cannot be found
+     */
+    private RegistryKey findRegistryKey(RegistryHiveFile registryHiveFile, String registryKey) {
+        
+        RegistryKey currentKey;
+        try {
+            RegistryKey rootKey = registryHiveFile.getRoot();
+            String regKeyList[] = registryKey.split("/");
+            currentKey = rootKey;
+            for (String key : regKeyList) {
+                currentKey = currentKey.getSubkey(key);
+            }
+        } catch (RegistryParseException ex) {
+            return null;
+        }
+        return currentKey;   
+
+    }
+    
+    /**
+     * Creates the attribute list for the given user information and group list.
+     *
+     * @param userInfo     Map of key\value pairs of user information
+     * @param groupList    List of the groups that user belongs
+     * @param existingUser
+     *
+     * @return List
+     *
+     * @throws ParseException
+     */
+    Collection<BlackboardAttribute> getAttributesForAccount(Map<String, String> userInfo, List<String> groupList, boolean existingUser, AbstractFile regAbstractFile) throws ParseException {
```

Comment:
Is this method used?

Reply:
This might be a merge issue, I removed this method in previous PR.  It is replaced by updateOsAccount

Reply:
It should be removed now.